Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -97,6 +97,15 @@ Create a custom command
 See the :ref:`Tutorial: Create a console command <console-command-tutorial>`
 for details on how to create commands.
 
+DataHandler usage
+=================
+Using the :ref:`DataHandler <tce-database-basics>` in a CLI command will
+fail with an error like this:
```

Comment:
Sorry, reading it again - maybe we should rephrase this:

```suggestion
maybe fail with an error, if no backend user authentication context has been added. In this case an error like this would be reported:
```